redi-connect-front/src/pages/app/profile/Profile.tsx

```diff
@@ -47,7 +47,7 @@ function Profile ({ loading, profile, currentUser, profilesFetchOneStart }: Prof
       {!loading && <LoggedIn>
         {(!isAcceptedMatch || currentUserIsMentor) && (
           <Button
-            onClick={() => history.push('/app/dashboard')}
+            onClick={() => history.goBack()}
```

Comment:
Entire PR looks good to me, my only doubt is this one. I don't see it really bringing any value, is there any issue changing the behaviour of this button to send the user "back" to any other page than the dashboard/applicant list?

Doing this change would make this brittle, cause we don't always know which chain of pages they've visited on the way into this page (though arguably it would most often be the applicant list page).

Reply:
Well the first thing you could not know is that the dashboard as is will be gone. The profile page can be shown in multiple contexts and the back button should actually always point back to where the user came from.

Reply:
Ah, ok. Cool. Approved then.

Reply:
Thanks for clarifying <3